spec/requests/food_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe 'Foods', type: :request do
+  describe 'GET /index' do
+    it 'should visit the home page' do
+      get '/'
+      expect(response).to render_template(:index)
+    end
+  end
+
+  describe 'GET /destroy' do
```

Comment:
I consider Destroy should be considered a POST method, not a GET. It would be better to change it so another developer can use the tests as documentation